unsplahAPI/unsplahAPI/Authentification/Authentication/View/AuthenticationViewController.swift

```diff
@@ -9,42 +9,161 @@ import UIKit
 import Combine
 
 class AuthenticationViewController: UIViewController, AuthenticationViewControllerProtocol {
-    
+
     var coordinator: AuthentificationCoordinatorProtocol?
     var viewModel: AuthenticationViewModelProtocol?
-    var userCancelable: AnyCancellable?
+    private var userCancelable: AnyCancellable?
+    private var unitOpacity: Float! {
+        didSet {
+            for button in buttonsStackView.arrangedSubviews as? [UIButton] ?? [] {
+                button.layer.opacity = unitOpacity
+            }
+            underButtonsLabel.layer.opacity = unitOpacity
+        }
+    }
+    @IBOutlet private weak var homeScreenView: UIView! {
+        didSet {
+            homeScreenView.backgroundColor = .white
+        }
+    }
+    @IBOutlet private weak var backgroundImage: UIImageView!
+    @IBOutlet private weak var backgroundImageTopConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var backgroundImageBottomConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var underImageView: UIView!
+    @IBOutlet private weak var buttonsStackView: UIStackView! {
+        didSet {
+            for button in buttonsStackView.arrangedSubviews as? [UIButton] ?? [] {
+                button.layer.masksToBounds = true
+                button.layer.cornerRadius = button.bounds.height / 4
+            }
+        }
+    }
+    @IBOutlet private weak var createAccountButton: UIButton! {
+        didSet {
+            createAccountButton.setAttributedTitle(
+                NSAttributedString(
+                    string: "Create Account",
+                    attributes: [
+                        NSAttributedString.Key.font: DesignedSystemFonts.bodyBold,
+                        NSAttributedString.Key.foregroundColor: DesignedSystemColors.primaryText
+                    ]),
+                for: .normal
+            )
+        }
+    }
+    @IBOutlet private weak var logInButton: UIButton! {
+        didSet {
+            logInButton.setAttributedTitle(
+                NSAttributedString(
+                    string: "Log In",
+                    attributes: [
+                        NSAttributedString.Key.font: DesignedSystemFonts.bodyBold,
+                        NSAttributedString.Key.foregroundColor: DesignedSystemColors.primaryText
+                    ]),
+                for: .normal
+            )
+        }
+    }
```

Comment:
Looks like these buttons are the same, let's create a custom button with following attributes and corner radius so we won't have duplicated code